test/xpu_api/tuple/tuple.tuple/tuple.cons/constexpr1_g.pass.cpp

```diff
@@ -0,0 +1,160 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/tuple>
+#include <oneapi/dpl/utility>
+
+#include "support/utils.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+// KSATODO constexpr_default_constructible
+struct constexpr_default_constructible
```

Comment:
This and the next helper structure are really strange for me. Can we replace it with more simple and straightforward constexpr function? E.g.

```
template <typename Tuple>
constexpr bool test_constexpr_default_ctor() {
    constexpr Tuple tpl;
    constexpr dpl::tuple_element_t<0, Tuple> desired_value;
    
    static_assert(dpl::get<0>(tpl) == desired_value);
    return true;
}
```

Reply:
Done